openmetadata-ui/src/main/resources/ui/src/pages/UserListPage/UserListPageV1.tsx

```diff
@@ -200,9 +200,9 @@ const UserListPageV1 = () => {
         // Converting string to URLSearchParameter
         const searchParameter = new URLSearchParams(location.search);
         // Getting the searched name
-        const searchTerm = searchParameter.get('search') || '';
-        setSearchValue(searchTerm);
-        getSearchedUsers(searchTerm, 1);
+        const userSearchTerm = searchParameter.get('userSearchTerm') || '';
```

Comment:
```suggestion
        const userSearchTerm = searchParameter.get('user') || '';
```
Let's make URL less cluttered

Reply:
Okay.

Reply:
@chirag-madlani , Done.